### DISCUSSION TITLE
Google FCM integration: Unexpected token I in JSON at position 0

### DISCUSSION THREAD

**Discussion by Volfor** [WalletConnect/walletconnect-monorepo#2783] [Developer Support]
Adding FCM Api key at the WalletConnect Cloud fails with an error: `Unexpected token I in JSON at position 0`
Could you help me with this?

![image](https://github.com/WalletConnect/walletconnect-monorepo/assets/5459173/e8b28f1c-54c8-4dd4-b714-4203c11de58e)

**Comment by Cali93** [comment]
Thanks for reporting this issue.
We are working on it and we will get back to you as soon as it has been fixed.
